.circleci/config.yml

```diff
@@ -65,6 +52,42 @@ jobs:
             npm config set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"
             npm publish
 
+  deploy-alpha:
+    <<: *container
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v2-uswds-dependencies-{{ checksum "package-lock.json" }}
+      - run: npm install --ignore-scripts
+      - run: npm rebuild node-sass
+      - run:
+          name: Build the USWDS package
+          command: npm run release
```

Comment:
@mejiaj Is the `release` command here the right one? Are we deploying with a fake hash key in the package every release?

Reply:
No, we shouldn't be. Does this task re-run creating the ZIP?

We should run the hash task once when we generate the ZIP we'll use to attach to GH release.

Reply:
I'm going to remove it from the job

Reply:
Changed this to `build`